bin/newver

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -eu
+
+if [ $# -eq 0 ]; then
+  >&2 echo "Usage: 'bin/newver X.Y.Z', where X.Y.Z is the Rubocop version to add."
+  exit 1
+fi
+NEWVER="$1"
+
+git checkout Gemfile*
+git checkout -b "newver/${NEWVER}"
+
+ed Gemfile <<-EDITS
+/gem "parser",/c
+gem "parser"
```

Comment:
is this replacing the line with the same line?

Reply:
It's replacing the version-specific line on `master` with one that doesn't specify a version. Master is pinned to an oddly-old parser, it seemed to me.